### ISSUE TITLE
How to reproduce the result for SciERC in the paper?

### ISSUE THREAD

**Issue by zplovekq** [Receiling/UniRE#5]
Thanks for your amazing work!
I do the data-preprocess and train as the guide in the Readme.md
I see the code set the seed for random、torch and np.
I train the model with the SciERC dataset and early stop at epoch 229 , get the result:
![image](https://user-images.githubusercontent.com/39880124/140073600-6b9d8846-08e8-48c1-b491-b5400c0fc988.png)
![image](https://user-images.githubusercontent.com/39880124/140072647-e3b19325-ab37-42b4-b5e8-3ff7dd9c80f3.png)
![image](https://user-images.githubusercontent.com/39880124/140072696-5e26c709-0d35-4a0c-8702-004153221f48.png)
I train it with a RTX3090 GPU without any modify the code (same as the Readme.md)
I see, in the paper, the result is:
![image](https://user-images.githubusercontent.com/39880124/140072874-da682af1-5a04-48b9-982d-ca20e0672052.png)
the F1 for the ent is 68.4 and i get 66.03
How to reproduce the result in the paper?
Sincerely looking forward to your reply!

**Comment by Receiling** [comment]
It is confusing that the second figure shows the entity F1 score is 69.67 rather than 66.03. 

As mentioned in the paper, all experiments are conducted on the Xeon(R) W-3175X CPU and Quadro RTX 8000 GPU.
To explore the impact of different machine conﬁguration, we retrain our model on the Tesla-V100-SXM2-32GB, the entity F1 is 67.86, the exact relation F1 is 35.77, which is indeed inferior to the reported result.
![image](https://user-images.githubusercontent.com/25247841/140247405-efb77081-83c5-4e51-a535-27e630d123a1.png)
![image](https://user-images.githubusercontent.com/25247841/140247316-b6a1da1d-88f9-457a-a039-5130ec851c18.png)

We recommend minor hyperparameter changes in different machine conﬁguration.
For example,tring different  random seed, more training epochs, different early stopping strategy.
In addition, you can also try to change the batch size from 32 to 16 or 64. 

If there are further questions about reproducing the results, please feel free to discuss with us.

**Comment by zplovekq** [comment]
Really Thanks to your apply.
Sorry..I screenshot the result uncorrectly...the 69.67 is the last valid, the real test score is :
![image](https://user-images.githubusercontent.com/39880124/140479764-bdfbdf69-4f81-4f9d-8a06-b507216c91cf.png)
![image](https://user-images.githubusercontent.com/39880124/140479646-71e3576e-d91b-4bf4-b96c-ab8f5b00885f.png)
In the above image, the ent f1 is 66.03(which is muchlower than 68.04) and the rel is 33.56 (also much lower than the 36.9)
So the answer for why the score is lower than that the paper said is the different GPU and CPU?
I will do some experiment on the my 3090....
So if i use different GPU and modify some hyperparameter , maybe it can improve the score? the hyperparameter and score should be different from different machine?
Sincerely looking forward for your reply~!

**Comment by Receiling** [comment]
The results on different GPU and CPU shows that different machine conﬁgurations do influence the final performance.
It is very possible that minor hyperparameter changes will lead to better results.
Note that just modifying few training hyperparameters, such as random seed, number of epochs, and batch size.

If there are further questions about reproducing the results, please feel free to discuss with us.

**Comment by Crescentz** [comment]
where is the file 'train.json'?
